traefik/Chart.yaml

```diff
@@ -1,6 +1,6 @@
 apiVersion: v1
 name: traefik
-version: 8.4.1
+version: 8.5.0
```

Comment:
There's another PR merged now, so I guess that would need to be 8.6.0?

Reply:
Updated to `8.6.0`